.github/ISSUE_TEMPLATE/documentation.yaml

```diff
@@ -0,0 +1,32 @@
+name: "📚 Documentation"
+description: "Report an issue related to documentation"
+title: "📚 Documentation: "
+labels: [documentation]
```

Comment:
Can you update this to the documentation label? ` 📖  Opportunity: Documentation`

Reply:
Applied ✌️